src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -888,20 +888,23 @@ public function schedule($params)
     {
         //Verify that the input time is valid
         if (!isset($params['time']) or !is_numeric($params['time'])) {
-            throw new MyRadioException('No valid Time was sent to the Scheduling Mapper.', MyRadioException::FATAL);
+            throw new MyRadioException(
+                'No valid Time was sent to the Scheduling Mapper.',
+                400
+            );
         }
         if ($params['time'] != -1 && !isset($this->requested_times[$params['time']])) {
             throw new MyRadioException(
                 'The Time value sent is not a valid Requested Time Reference.',
-                MyRadioException::FATAL
+                500
```

Comment:
Why is this one a 500?

Pretty sure none of these exceptions need to be split over multiple lines either

Reply:
The multiple lines thing was due to 120 char line length.
Could be a 400, wasn't sure given it was once a "FATAL" error.

Reply:
All the others were FATAL as well :p
The only line that's gotten longer is the last one, all the otherse are shorter than they were

Reply:
I think the issue was it was ignoring the lengths, until I changed them and they were still over 120chars. I've changed the 500 now though.